Detections/Corelight/CorelightNetworkServiceScanning.yaml

```diff
@@ -0,0 +1,33 @@
+id: 599570d4-06f8-4939-8e29-95cd003f1abd
+name: Corelight - Network Service Scanning Multiple IPs

```

Comment:
Instead of IPS use IP Addresses.

Thanks,
Aditi Maheshwari

Reply:
updated